src/axom/core/Path.cpp

```diff
@@ -45,12 +62,29 @@ Path Path::join(std::initializer_list<Path> paths, const char delim)
               path.m_components.end(),
               std::back_inserter(result.m_components));
   }
+
+  if(paths.size() != 0 && (*paths.begin()).m_leading_delim)
+  {
+    result.m_leading_delim = true;
+  }
+  else
+  {
+    result.m_leading_delim = false;
+  }
+
   return result;
 }
 
 Path::operator std::string() const
 {
-  return fmt::format("{0}", fmt::join(m_components, std::string(1, m_delim)));
+  std::string result;
+  if(m_leading_delim)
+  {
+    result = "/";
```

Comment:
Should this account for a non-default delimiter?
E.g.
```suggestion
    result = m_delim;
```

Reply:
Thanks for catching that, I overlooked this one.